CMakeLists.txt

```diff
@@ -53,6 +53,6 @@ endif()
 ### Applications
 ###
 
-if(BUILD_TESTING AND QTRANSPORT_BUILD_TESTS)
+#if(BUILD_TESTING AND QTRANSPORT_BUILD_TESTS)
   add_subdirectory(cmd)
-endif()
+#endif()
```

Comment:
Would prefer we not build what we don't need in other projects where this is included.